docs/settings.md

```diff
@@ -0,0 +1,60 @@
+# Settings
+
+Library and connection settings.
+
+1. To create the connection please follow the example below:
+
+```typescript
+import * as users from './dummy/usersModel';
+import { Engines, Drivers } from '@unicoderns/orm';
+
+let usersTable: users.Users;
+usersTable = new users.Users({
+  debug: false,
+  engine: Engines.MySQL,
+  driver: Drivers.Native,
+});
+```
+
+You should create the model you need, in the example above we created the model `user` and we imported it in the class we need. Please refer to [models](https://github.com/unicoderns/orm/blob/master/docs/models.md) in order to know how you can create a model.
+
+Then, once you instance the model you should pass the required parameters to create the connection:
+
+`debug: boolean`
+`engine: Engines | .MySQL | .PostgreSQL`
```

Comment:
There's a reason to describe them like this? What if we use Engines [ .Mysq l .PostgreSQL ]? Do you think it may be better?

Reply:
There's no particular reason. I think it's better the way you proposed.